src/r2mm/mods/ModBridge.ts

```diff
@@ -27,10 +32,45 @@ export default class ModBridge {
         const mod: ManifestV2 = new ManifestV2().fromReactive(vueMod);
         const latestVersion: ThunderstoreVersion | void = ModBridge.getLatestVersion(mod, ThunderstorePackages.PACKAGES);
         if (latestVersion instanceof ThunderstoreVersion) {
-            return mod.getVersionNumber()
-                .isEqualTo(latestVersion.getVersionNumber()) || mod.getVersionNumber().isNewerThan(latestVersion.getVersionNumber());
+            return mod.getVersionNumber().isEqualOrNewerThan(latestVersion.getVersionNumber());
         }
         return true;
     }
```

Comment:
Could this entire function be removed? Doesn't seem to be used outside of a single place now